client/jetpack-cloud/style.scss

```diff
@@ -318,8 +318,8 @@ html {
 	}
 }
 
-.site-selector {
-	display: block;
```

Comment:
@danielbachhuber was this change intentional? I'm treating the `.site-selector` as a block element instead of a flexible one because then the "Add new site" button would get close to the site list. This was the behavior we had before this PR, so I thought of maintaining it.

| Right now | How it was |
| ---------- | ------- |
| ![Screen Shot 2022-09-08 at 17 28 31](https://user-images.githubusercontent.com/26530524/189219851-ee373f45-ccd8-4a63-997a-4e619140daec.png) | ![Screen Shot 2022-09-08 at 17 29 39](https://user-images.githubusercontent.com/26530524/189219890-507cfa0f-76cb-41e7-93c9-42039028fb65.png) |

Reply:
@zaguiini Yep, it was: 83ec747f385c37b612a4785f076d664f396c94fe It was @p-jackson's preference.

Reply:
This project has a goal of not breaking Jetpack Cloud, and I think that's a balance. It doesn't mean _no_ changes at all to JP Cloud IMO.
Jetpack Cloud is a little different from regular Calypso, but I don't think it should needlessly diverge.